CONTRIBUTING.md

```diff
@@ -41,12 +42,12 @@ Getting your contributions integrated is relatively straightforward, here
 is the checklist:
 
 * All tests pass
-* Consensus is reached. This basically mean that the reviewers add `LGTM` comments. `LGTM` is a
+* Consensus is reached. This basically means that the reviewers add `LGTM` comments. `LGTM` is a
```

Comment:
Can you clarify to say that there should be at least one `LGTM`?  Right now it is is a little more vague--"reviewers add `LGTM` comments."--how many?